### PR TITLE
closes #238 - Display tutorial for Arango 

### LINKED ISSUE #238: Tutorial for arango database
The websites needs to display a tutorial, as well as all the functionalities needed in that tutorial for Arango

### PR DESCRIPTION
**Pull request by anthonykhoa**
The websites needs to display a tutorial, as well as all the ability to create a database for Arango. 
This PR **cannot** be merged until we have a production arango database.
This PR will:
* Include a tutorial for Arango
* Update relevant back-end files needed to enable `create my database` button for on the Arango tutorial page
* Update tutorial to display `an` instead of `a` before database names that start with a vowel
Screen Shot 2020-10-07 at 11 44 41 AM
Screen Shot 2020-10-07 at 11 25 20 AM
Screen Shot 2020-10-07 at 11 25 41 AM
Screen Shot 2020-10-07 at 11 25 47 AM

### CODE REVIEW

File: src/routes/renderRoutes.js [hunk 1]
```diff
@@ -1,41 +1,36 @@
 const db = require("../../sequelize/db");
 const es = require("../../database/elasticsearch/elastic");
 const pg = require("../../database/postgres/pg");
-
+const arangoModule = require("../../database/arango/arango");
+require("dotenv").config();
 const routes = {};
 
+// This is the 'host' url for a person's database credentials
+const dbHost = {
```
Comment (Ulisseus) [inline src/routes/renderRoutes.js:9]: I believe postgres uses `process.env.HOST` for sequelize and elastic uses `process.env.ES_HOST`.

Comment (Ulisseus) [inline src/routes/renderRoutes.js:9]: Shouldn't it be `process.env.HOST` for postgres/sequelize and `process.env.ES_HOST` for elastic?

File: src/routes/renderRoutes.js [hunk 2]
```diff
@@ -1,41 +1,36 @@
...
+  Postgres: "learndatabases.dev",
+  Elasticsearch: "elastic.learndatabases.dev",
+  Arango: process.env.ARANGO_URL,
+};
+
+const checkAccount = {
+  Postgres: ({ username }) => pg.userHasPgAccount(username),
+  Elasticsearch: (user) => es.checkAccount(user),
+  Arango: ({ username }) => arangoModule.checkIfDatabaseExists(username),
+};
```
Comment (Ulisseus) [inline src/routes/renderRoutes.js:19]: Shouldn't it be `process.env.HOST` for postgres/sequelize and `process.env.ES_HOST` for elastic?

File: src/routes/renderRoutes.js [hunk 3]
```diff
@@ -1,41 +1,36 @@
...
+const arangoModule = require("../../database/arango/arango");
+require("dotenv").config();
 const routes = {};
 
+// This is the 'host' url for a person's database credentials
+const dbHost = {
+  Postgres: "learndatabases.dev",
+  Elasticsearch: "elastic.learndatabases.dev",
+  Arango: process.env.ARANGO_URL,
+};
```
Comment (Ulisseus) [inline src/routes/renderRoutes.js:13]: Shouldn't it be `process.env.HOST` for postgres/sequelize and `process.env.ES_HOST` for elastic?

Comment (anthonykhoa) [reply src/routes/renderRoutes.js:13]: You are right -- I simply copy and pasted the links used for `dbHost` that the last person that touched this file used. It should be `process.env.ES_HOST` for elastic, and the testing files should change to reflect that too. For Postgres, I am not sure. It seems we just use `127.0.0.1` as our production `HOST` env variable.

**Review by songz — APPROVED** [review]

**Review by Ulisseus — APPROVED** [review]

**Pull request merged**